lib/elrondex/sc/pair_sc.ex

```diff
@@ -1,6 +1,17 @@
 defmodule Elrondex.Sc.PairSc do
   alias Elrondex.{Sc, Transaction, Account, REST, ESDT, Pair, Network}
 
+  @doc """
+  Swaps token1 amount for a minimum of token2 amount.
+
+  ## Arguments
+    * `account` - An elrond account (as described in lib/account)
+    * `pair` -  An elrond pair (as described in lib/pair)
+    * `token_in` - Token_in to be swapped
+    * `value_in` - Token_in amount to be swapped
```

Comment:
I think you can find a better format not just write 'token_in' eg: "First token"